target/linux/ath79/dts/qca9531_comfast_cf-wr752ac.dts

```diff
@@ -0,0 +1,117 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca953x.dtsi"
+
+/{
+    compatible = "comfast,cf-wr752ac", "qca,qca9531";
+	model = "COMFAST CF-WR752AC";
```

Comment:
Please care about proper indent in the entire file.

Reply:
Thanks for taking some time to review my code.
I think I fix all the idents now.